tests/test_wsi_registration.py

```diff
@@ -0,0 +1,55 @@
+import urllib
+
+import numpy as np
+import PIL.Image as Image
+import pytest
+
+from tiatoolbox.tools.registration.wsi_registration import DFBRegistrtation
+
+
+def test_extract_features():
+    """Test for CNN based feature extraction function."""
+    main_url = "https://tiatoolbox.dcs.warwick.ac.uk/testdata/registration/"
+    urllib.request.urlretrieve(main_url + "HE_1_level8_gray.png", "sample.png")
```

Comment:
Please use `tmp_path` and `conftest.py`. Please see links below for examples.

https://github.com/TissueImageAnalytics/tiatoolbox/blob/af7319ef633e2f8b315b0949ed343f67d38bab0a/tests/conftest.py#L52

https://github.com/TissueImageAnalytics/tiatoolbox/blob/af7319ef633e2f8b315b0949ed343f67d38bab0a/tests/test_wsimeta.py#L84

Reply:
Added to remote_samples files:
 https://github.com/TissueImageAnalytics/tiatoolbox/blob/e89c666bf952ab1f0854cf1c5e7679e48b614e2d/tiatoolbox/data/remote_samples.yaml#L112-L115